apps/server/src/modules/account/api/account.uc.spec.ts

```diff
@@ -291,7 +291,7 @@ describe('AccountUc', () => {
 			it('should return one account', async () => {
 				const { mockSuperheroUser, mockStudentUser, mockStudentAccount } = setup();
 				const accounts = await accountUc.searchAccounts(
-					{ userId: mockSuperheroUser.id } as ICurrentUser,
+					iCurrentUserFactory.buildWithId({ userId: mockSuperheroUser.id }),
 					{ type: AccountSearchType.USER_ID, value: mockStudentUser.id } as AccountSearchDto
 				);
 				const expected = new ResolvedSearchListAccountDto(
```

Comment:
`build` is enough as ICurrentUser does not have an id.
```suggestion
				const accounts = await accountUc.searchAccounts(iCurrentUserFactory.build({ userId: mockSuperheroUser.id }), {
```